docs/labels.md

```diff
@@ -76,3 +76,7 @@ The underlying type of the workload supported by the preference, current values
 ### `instancetype.kubevirt.io/arch`
 
 The underlying architecture of the workload supported by the preference, current values are `arm64` or `amd64`.
+
+### `instancetype.kubevirt.io/unsupported`
+
+The preference is not supported.
```

Comment:
What does it actually mean? Who supports preferences at all?

Reply:
Dropped.